using System;

namespace Lektion_8_Test
{
    class Program
    {
        static void Main(string[] args)
        {
            //Opgave 1 Explain, in your own words, the concept of exceptions.
            // Det er når der sket noget som man ikke regner med og derefter skal håntere det på en speciel måde

            //Opgave 2 Explain, in your own words, the concept of code coverage.
            // Hvor stor en del af koden som bliver dækket af tests

            //Opgave 3 Explain, in your own words, the process of test-driven development.
            // Hver gang man har lavet en funktion eller en del af programmet, så laver man nogle testss til denne del præcist

            //Opgave 4
            //Circle c = new Circle(new Tuple<double, double>(2, 4),  5);
            //Console.WriteLine(c.IsInsideCircle(new Tuple<double, double>(2, 5)));
            //Console.WriteLine(c.CircleOverlapping(new Tuple<double, double>(1, 3), 1));

            //Opgave 5

            

        }
    }
}
